spec/functional/resource/group_spec.rb

```diff
@@ -169,8 +173,11 @@ def create_group
 
     describe "when the users exist" do
       before do
+        high_uid = 60000
```

Comment:
little worried that could interact badly/surprisingly with systems that have 32-bit signed ints for uids.   i can't recall when linux switched.  no idea when/if solaris or aix is off of those.  dunno...